language/e2e-tests/src/tests/genesis.rs

```diff
@@ -36,3 +37,23 @@ fn invalid_genesis_write_set() {
         VMStatus::new(StatusCode::INVALID_WRITE_SET)
     );
 }
+
+#[test]
+fn execute_genesis_write_set() {
+    let executor = FakeExecutor::no_genesis();
+    // Genesis write sets are not allowed to contain deletions.
+    let write_set = GENESIS_WRITE_SET.clone();
+    let address = account_config::association_address();
+    let (private_key, public_key) = compat::generate_keypair(None);
```

Comment:
Please avoid using the compat interface as it is deprecated.

Reply:
Victor's `KeyGen` is a great alternative to this that's used elsewhere in e2e tests